### PR TITLE
feat(core-select): add hintPosition below

### PR DESCRIPTION
**Pull request by jraff**
## Related issues

See #1131 

## Checklist before submitting pull request

- [ ] New code is unit tested
- Commits follow our [Developer Guide](https://tds.telus.com/contributing/developer-guide.html#make-a-commit)
- [ ] For code changes, run `npm run prepr` locally
  - make sure visual and accessibility tests pass

### CODE REVIEW

File: packages/Select/Select.jsx [hunk 1]
```diff
@@ -131,18 +131,21 @@ const renderError = (error, errorId) => (
...
-        <Text size="medium" bold data-testid="selectLabel">
-          {label}
-        </Text>
-        {hint && renderHint(hint, Text, hintId)}
-      </StyledLabelContainer>
-    </label>
-    {tooltip && React.cloneElement(tooltip, { connectedFieldLabel: label })}
-  </Box>
+const renderLabel = (id, label, hint, hintPosition, hintId, tooltip) => (
+  <div>
```
Comment (agorovyi) [inline packages/Select/Select.jsx]: Can this `<div>` be replaced with Fragment?

File: packages/Select/Select.jsx [hunk 2]
```diff
@@ -279,6 +283,10 @@ Select.propTypes = {
    * Clarify the expected input.
    */
   hint: PropTypes.string,
+  /**
+   * Position of the `hint` relative to `label`.
+   */
```
Comment (agorovyi) [inline packages/Select/Select.jsx]: Could you also add `@since x.x.x` and specify which version this prop is available since?

Comment (jraff) [reply packages/Select/Select.jsx]: Added.

File: packages/Select/Select.jsx (hunk 1 above)
Comment (jraff) [reply packages/Select/Select.jsx]: No, it needs to use the `div` so as to not be affected by the `Box`

**Pull request merged**